doc/man/8/radosgw-admin.rst

```diff
@@ -455,20 +814,16 @@ Quota Options
 
 .. option:: --max-size
 
-	Specify max size (in bytes, negative value to disable).
+	Specify max size (negative value to disable).
 
 .. option:: --quota-scope
 
-	Scope of quota (bucket, user).
+	The scope of quota (bucket, user).
 
 
 Orphans Search Options
 ======================
 
-.. option:: --pool
```

Comment:
This is a option used by orphans find

Reply:
Please see L535 in the changed file. There were two `--pool`s and I have merged them into one. If you want I can revert. Please suggest.

Reply:
you can keep it

Reply:
Done. Please verify.